packages/patternfly-3/patternfly-react-extensions/less/catalog-tile.less

```diff
@@ -0,0 +1,65 @@
+.catalog-tile-pf {
+  background: @card-pf-bg-color;
+  border: 1px solid @color-pf-black-300;
+  box-shadow: 0 1px 1px rgba(3, 3, 3, 0.175);
+  height: @catalog-tile-pf-height;
+  margin: 0 20px @catalog-tile-pf-margin-bottom 0;
+  padding: 15px;
+  width: @catalog-tile-pf-width;
+
+  &.featured {
+    border-top-color: @card-pf-accented-border-top-color;
+    border-top-width: 2px;
+  }
+}
+
+.catalog-tile-pf-header {
+  display: flex;
+  font-size: 16px;
+  font-weight: 400;
+  padding: 0;
```

Comment:
is this necessary?

Reply:
nope, i was extending the card-pf originally until it was just too different and I was overriding just about all the settings.